src/main/java/com/gitblit/wicket/pages/EditUserPage.java

```diff
@@ -106,8 +106,21 @@ protected void setupPage(final UserModel userModel) {
 		final List<RegistrantAccessPermission> permissions = app().repositories().getUserAccessPermissions(userModel);
 
 		final Palette<String> teams = new Palette<String>("teams", new ListModel<String>(
-				new ArrayList<String>(userTeams)), new CollectionModel<String>(app().users()
-				.getAllTeamNames()), new StringChoiceRenderer(), 10, false);
+				new ArrayList<String>(userTeams)), new CollectionModel<String>(app().users().getAllTeamNames()), 

```

Comment:
Assuming this change is necessary, I would expect we'd need it anywhere we use a palette for teams.